javascript-functions/main.js

```diff
@@ -0,0 +1,38 @@
+function convertMinutesToseconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+
+console.log(convertMinutesToseconds(5));
+
+function greet(name) {
+  var greeting = 'hello' + ' ' + name;
+  return greeting;
+}
+
+console.log(greet('sal'));
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+
+console.log(getArea(5, 4));
+
+function getFirstName(person) {
+  var name = person.firstName;
+  return name;
+}
+var object = {
+  firstName: 'Cash',
+  lastName: 'Money'
+};
+
+console.log(getFirstName(object));
+
+function getLastElement(array) {
+  var last = array.length;
```

Comment:
this is close, but you don't want the length of the array, you want to return the last item in the array.

Reply:
![Screenshot (117)](https://user-images.githubusercontent.com/51901536/130675375-b9a1efa0-825f-4153-a873-8a4e9e2017a5.png)